assets/styles/app.scss

```diff
@@ -27,16 +27,112 @@ $animCurveEaseOut: cubic-bezier(.1, 1, .8, .8);
 @import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;900&family=Staatliches&display=swap');
 
 :root {
-
     --plyr-color-main: #581EA3;
+    --plyr-range-thumb-background: #FFFFFF;
+    --plyr-control-icon-size: 20px;
+    --plyr-video-control-color: #FFFFFF;
+    --plyr-video-control-color-hover: #FFE500;
+}
 
-    --plyr-range-thumb-background: #ffffff;
+// header ----------------------------------------------------
+
+.home-image {
+    background-image: url("/assets/images/poster1-LOL.jpg");
+    background-size: cover;
+    margin: auto;
+    width: 80vw;
+    min-width: 300px;
+    min-height: 300px;
+    object-fit: cover;
+    background-position: center;
+
+    &::after {
+        content: "";
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        width: 100%;
+        height: 25%;
+        background-image: linear-gradient(to bottom, transparent, $secondary);
+    }
+}
 
-    --plyr-control-icon-size: 20px;
+.home-title {
+    height: 40px;
+    width: 80vw;
+    font-size: $h4-font-size;
+}
 
-    --plyr-video-control-color: #ffffff;
+.header-logo {
+    background-color: $dark;
+    font-size: 45px;
+    border-radius: 0 0 8px 0;
+}
 
-    --plyr-video-control-color-hover: #FFE500;
+.button-view {
+    min-height: 300px;
+    margin-bottom: 20px;
+    z-index: 1;
+
+    .link-desktop {
+        background: $primary !important;
+        font-size: $h3-font-size;
+        text-decoration: none;
+        width: 200px;
+        padding: 5px;
+
+
+        .view {
+            margin-left: 20px;
+        }
+
+        .icon {
+            margin-right: 10px;
+            font-size: 35px;
+            margin-left: 10px;
+
+        }
+    }
+
+    .link-mobile {
+        background: $primary !important;
+        font-size: $h6-font-size;
+        text-decoration: none;
+        width: calc(0.8*200px);
+        padding: 6px 3px;
```

Comment:
Should be done with Bootstrap.
bg-primary
fs-6
text-decoration-none
//
py-1,2,3,4,5
px-1,2,3,4,5